modules/install-sno-generate-the-discovery-iso-manually.adoc

```diff
@@ -0,0 +1,152 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso-manually_{context}"]
+= Generate the discovery ISO manually
+
+Installing {product-title} on a single node requires a discovery ISO, which you can generate with the following procedure.
+
+.Procedure
+
+. Download the {product-title} client (`oc`) and make it available for use by entering the following command:
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz > oc.tar.gz
+----
++
+[source,terminal]
+----
+$ tar zxf oc.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x oc
+----
```

Comment:
Unsure why we need `oc` here, we don't use it

Reply:
This is true. However, i guess it's good to have it to check the cluster after install? We have this as a step later: 
https://deploy-preview-39205--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-installing-sno.html#monitoring-the-installation-manually_install-sno-installing-sno-with-the-assisted-installer